docs/reference/elasticsearch/roles.md

```diff
@@ -1,20 +1,28 @@
 ---
 mapped_pages:
   - https://www.elastic.co/guide/en/elasticsearch/reference/current/built-in-roles.html
+applies_to:
+  stack: all
 ---
 
 # Roles [built-in-roles]
 
-:::{note}
 This section provides detailed **reference information** for Elasticsearch privileges.
 
-Refer to [User roles](docs-content://deploy-manage/users-roles/cluster-or-deployment-auth/user-roles.md) in the **Deploy and manage** section for overview, getting started and conceptual information.
-:::
+If you're using a stack-versioned deployment such as a self-managed cluster, {{ech}}, {{eck}}, or {{ece}}, then refer to [User roles](docs-content://deploy-manage/users-roles/cluster-or-deployment-auth/user-roles.md) for more information on how role-based access control works.
```

Comment:
no need to clarify "if you're using stack" here because the page is labeled as stack and serverless has different built-in roles

if you really felt fancy you could add a note to that effect (details [here](https://www.elastic.co/docs/deploy-manage/users-roles/cloud-organization/user-roles#general-assign-user-roles-table))

Reply:
Thing is I wanted to include another link to info about roles in Serverless (similar to what we did in the privileges page), and then I forgot to get back to that paragraph and add the link. 

Do you think it works now, @shainaraskas ? 
I've isolated it in a tip because it only slightly relates to the content on the page, but could be very useful to someone who landed on that page accidentally and was looking for directions on how to get to the **Deploy and manage** section.